src/students/dto/page-meta.dto.ts

```diff
@@ -0,0 +1,31 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { PageMetaDtoParameters } from 'src/types';
+
+export class PageMetaDto {
+  @ApiProperty()
```

Comment:
Czy tutaj potrzebujemy swaggera? czy ja czegoś nie rozumiem?:)

inna sprawa, to wszystko jest okej :) ale zawsze można spróbować zrobić to ładniej :)
Spróbuj zrobić tak (testowo i sprawdź czy Ci zadziała tak jak teraz)
```js
import { Transform } from 'class-transformer';

export class PageMetaDto {
  readonly page: number;

  readonly take: number;

  readonly itemCount: number;

  @Transform(({ obj }) => Math.ceil(obj.itemCount / obj.take))
  readonly pageCount: number;

  @Transform(({ obj }) => obj.page > 1)
  readonly hasPreviousPage: boolean;

  @Transform(({ obj }) => obj.page < obj.pageCount)
  readonly hasNextPage: boolean;
}
```
a potem w metodzie do pobierania:
```js
public async getAllStudents(pageOptionsDto: PageOptionsDto){
...
const pageMetaDto = plainToInstance(PageMetaDto, { ...pageOptionsDto, itemCount });

  return new PageDto(students, pageMetaDto);
  }
  ```